src/System.Device.Ports/System.Device.Ports.SerialPort/Parity.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Device.Ports.SerialPort
+{
+    /// <summary>
+    /// Specifies the parity bit for a SerialPort object.
+    /// </summary>
+    public enum Parity
+    {
+        /// <summary>
+        /// No parity check occurs.
```

Comment:
This description is wrong. Right would be: No parity bit is sent or expected.

Reply:
Good catch, as I said many descriptions from the official docs are inaccurate.